### PR TITLE
Update mocha-related packages to remove NSP warnings about "growl"

### PR DESCRIPTION
**Pull request by nickpape-msft**
We are using old versions of several mocha-related packages. These are using the dependency "growl" which is getting flagged by our system as having a vulnerability [however, we don't believe there is a vulnerability that actually affects us]. The easiest way to get rid of the warning is to upgrade mocha, and by extension, "growl".

### CODE REVIEW

File: stack/rush-stack/gulpfile.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 'use strict';
 
 const build = require('@microsoft/node-library-build');
-
+build.mocha.enabled = false;
```
Comment (iclanton) [inline stack/rush-stack/gulpfile.js]: Intentional?

File: webpack/resolve-chunk-plugin/gulpfile.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 'use strict';
 
 let build = require('@microsoft/node-library-build');
-
+build.mocha.enabled = false;
```
Comment (iclanton) [inline webpack/resolve-chunk-plugin/gulpfile.js]: Same with these.

**Comment by iclanton** [comment]
Approved with suggestions.

File: stack/rush-stack/gulpfile.js (hunk 1 above)
Comment (nickpape-msft) [reply stack/rush-stack/gulpfile.js]: Yep, these projects don't have tests and the new mocha throws if you don't have any tests..

File: webpack/resolve-chunk-plugin/gulpfile.js (hunk 2 above)
Comment (nickpape-msft) [reply webpack/resolve-chunk-plugin/gulpfile.js]: See other comment

**Pull request merged**